python/ray/worker.py

```diff
@@ -264,6 +262,11 @@ def put_object(self, value, object_id=None, pin_object=True):
                 "do this, you can wrap the ray.ObjectID in a list and "
                 "call 'put' on it (or return it).")
 
+        if self.mode == LOCAL_MODE:
+            # TODO(ilr): Figure out how to make this work
```

Comment:
I wouldn't worry about figuring out how to support it. It isn't a public API and we should probably remove all uses of it internally as well.

Reply:
Okay, I'll drop this todo